packages/terra-site/src/App.jsx

```diff
@@ -164,11 +174,14 @@ class App extends React.Component {
 
     // Moved Base to wrap the main content, as i18nProvider inserts an unstyled div that ruins layout if placed higher.
     // Might consider enablling styling for Base, or evaluate if multipe Bases are viable.
+    const theme = this.state.theme.split(' ');
     const mainContent = (
-      <ThemeProvider themeName={this.state.theme}>
-        <Base id="site-content-section" className={styles['site-content']} locale={this.state.locale}>
-          {this.props.children}
-        </Base>
+      <ThemeProvider themeName={theme[0]}>
+        <ThemeProvider themeName={theme[1]}>
```

Comment:
how does this behave when you pass in a null theme name?

Reply:
The themeName prop is optional so it looks to work if there is only a top-level theme and no sub theme.

Heres a screenshot for react-dev-tools with the top level mock-theme selected
<img width="1430" alt="screen shot 2017-08-29 at 1 18 25 pm" src="https://user-images.githubusercontent.com/633148/29837051-b96bb73c-8cbc-11e7-8bc8-f32255d4a6df.png">

No console errors were thrown.